libs/pages/database/src/lib/ui/page-settings-general/page-settings-general.tsx

```diff
@@ -166,30 +171,75 @@ export function PageSettingsGeneral({
                   )}
                 </>
               )}
-              <Controller
-                name="accessibility"
-                control={control}
-                rules={{ required: true }}
-                render={({ field, fieldState: { error } }) => (
-                  <div>
-                    <InputSelect
-                      dataTestId="input-select-accessibility"
-                      label="Accessibility"
-                      options={databasesAccessibility}
-                      onChange={field.onChange}
-                      value={field.value}
-                      error={error?.message}
-                    />
-                    <ExternalLink
-                      className="ml-4 gap-0.5"
-                      href="https://hub.qovery.com/docs/using-qovery/configuration/database/#accessibility"
-                      size="xs"
-                    >
-                      Learn more
-                    </ExternalLink>
-                  </div>
-                )}
-              />
+              {publicOptionNotAvailable ? (
+                <span>
+                  The access of your database is private, it is only accessible from within your cluster or via our
+                  port-forward feature. Public access to a K3S cluster running a containerized database is not
+                  supported.
+                </span>
+              ) : (
+                <Controller
+                  name="accessibility"
+                  control={control}
+                  rules={{ required: true }}
+                  render={({ field }) => (
+                    <div>
+                      <SegmentedControl.Root
+                        defaultValue={DatabaseAccessibilityEnum.PRIVATE}
+                        onValueChange={field.onChange}
+                        value={field.value}
+                        className="w-60 text-sm"
+                      >
+                        <SegmentedControl.Item value={DatabaseAccessibilityEnum.PRIVATE}>
+                          Private access
+                        </SegmentedControl.Item>
+                        <SegmentedControl.Item value={DatabaseAccessibilityEnum.PUBLIC}>
+                          Public access
+                        </SegmentedControl.Item>
+                      </SegmentedControl.Root>
+                      <p className="mt-2 text-sm text-neutral-350">
+                        {match({ watchMode, watchAccessibility })
+                          .with(
+                            { watchMode: 'CONTAINER', watchAccessibility: 'PRIVATE' },
+                            { watchMode: 'CONTAINER', watchAccessibility: undefined },
+                            () => (
+                              <>
+                                <strong>Private access to your database is ensured</strong>, as it is only accessible
```

Comment:
it's not strong but `medium` should be a `<span className="font-medium">`

Reply:
fixed https://github.com/Qovery/console/pull/1651/commits/a11b475575c43e62269a4dd3cdc5bbef3bb6f020